homeassistant/components/senz/climate.py

```diff
@@ -0,0 +1,104 @@
+"""nVent RAYCHEM SENZ climate platform."""
+from __future__ import annotations
+
+from typing import Any
+
+from aiosenz import MODE_AUTO, Thermostat
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, PRECISION_TENTHS, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import SENZDataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the SENZ climate entities from a config entry."""
+    coordinator: SENZDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        SENZClimate(thermostat, coordinator) for thermostat in coordinator.data.values()
+    )
+
+
+class SENZClimate(CoordinatorEntity, ClimateEntity):
+    """Representation of a SENZ climate entity."""
+
+    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_precision = PRECISION_TENTHS
+    _attr_hvac_modes = [HVAC_MODE_HEAT, HVAC_MODE_AUTO]
+    _attr_supported_features = SUPPORT_TARGET_TEMPERATURE
```

Comment:
This should use `ClimateEntityFeature. TARGET_TEMPERATURE ` <https://developers.home-assistant.io/blog/2022/04/02/support-constants-deprecation/>